module/storage.rb

```diff
@@ -1,11 +1,22 @@
 require 'json'
+require 'fileutils'
 require_relative '../lib/book'
 require_relative '../lib/label'
 require_relative '../lib/genre'
 require_relative '../lib/game'
 require_relative '../lib/author'
 
 module Storage
+  def check_data_files
```

Comment:
[OPTIONAL]
Instead of writing a lot of lines, how about adding 
`File.new(file_path, 'w+') unless File.exist?(file_path)`

this kind of way in fetch_data. Instead of this line in line 90
`return nil if File.empty?(file_path)`

And also you should remove the require 'fileutils' if you were to use this way